zkevm-circuits/src/super_circuit.rs

```diff
@@ -600,7 +641,17 @@ impl<
         let state_circuit = StateCircuit::new_from_block(block);
         let tx_circuit = TxCircuit::new_from_block(block);
         let pi_circuit = PiCircuit::new_from_block(block);
+        // let bytecode_circuit = BytecodeCircuit::new_from_block(block);
+        // let bytecode_circuit1 = BytecodeCircuit::new_from_block(block);
+        // Get each sub circuit's bytecodes and assign
+        #[cfg(feature = "dual_bytecode")]
+        let bytecode_circuit = BytecodeCircuit::new_from_block_for_dual_circuit(block, true);
+        #[cfg(feature = "dual_bytecode")]
+        let bytecode_circuit1 = BytecodeCircuit::new_from_block_for_dual_circuit(block, false);
+
+        #[cfg(not(feature = "dual_bytecode"))]
```

Comment:
remeber in super_circuit.rs, update `min_num_rows_block_subcircuits`.  Don't keep 2 elem for bytecode circuits, but use the max among them.

Reply:
good point, will check it.

Reply:
modifed BytecodeCircuit::<Fr>::min_num_rows_block to return max rows of two sub circuit in [f2a35a2](https://github.com/scroll-tech/zkevm-circuits/pull/1368/commits/f2a35a2f55d5776aa07eb8d54aa59afa2a81b635)